src/basic/confidential-virt.c

```diff
@@ -226,7 +226,18 @@ static ConfidentialVirtualization detect_confidential_virtualization_impl(void)
 
         return CONFIDENTIAL_VIRTUALIZATION_NONE;
 }
+#elif defined(__aarch64__)
+static ConfidentialVirtualization detect_confidential_virtualization_impl(void) {
+        int r;
 
+        r = access("/sys/devices/platform/arm-cca-dev", F_OK);
```

Comment:
RET_NERRNO

Reply:
Sure.

Reply:
That also explains why I was getting a permission denied rather than no such directory when I was testing this, seemed odd at the time.